src/keycloak_utils/consumer/rest_framework/core.py

```diff
@@ -0,0 +1,288 @@
+import socket
+import threading
+from functools import partial, singledispatch
+import msgpack
+import logging
+import time
+import pika
+from pika.exceptions import (
+    AMQPConnectionError,
+    AMQPChannelError,
+    ConnectionClosedByBroker,
+)
+from ...contrib.django.conf import (
+    KC_UTILS_CREATE_QUEUES,
+    KC_UTILS_SYNC_QUEUES,
+    RABBITMQ_URL,
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class KeycloakEventHandler:
+    @staticmethod
+    def process_message(event_data):
+        from .strategies import EventTypeStrategyClassFactory
+
+        try:
```

Comment:
huge try and except block, this is a super bad practice, when you are unsure of something can go wrong, you wrap everything in a try/exc block. thats not the way. every potential exception of your code must be handled individually